components/forum/ForumReply.js

```diff
@@ -39,20 +50,20 @@ export default function ForumReply({
   )
   const allRepliesHidden = allChildIds.every((childId) => displayOptionsMap[childId].hidden)
 
-  const showReplyInvitations =
-    note.replyInvitations?.filter((i) => !excludedInvitations?.includes(i.id)).length > 0 &&
-    !note.ddate
-
-  const scrollToNote = (noteId, showEditor) => {
-    const el = document.querySelector(
-      `.note[data-id="${noteId}"]${showEditor ? ' .invitation-buttons' : ''}`
-    )
-    if (!el) return
-
-    const navBarHeight = 63
-    const y = el.getBoundingClientRect().top + window.pageYOffset - navBarHeight
-
-    window.scrollTo({ top: y, behavior: 'smooth' })
+  let replyInvitations = []
+  if (!note.ddate) {
+    replyInvitations = note.replyInvitations?.filter((inv) => {
+      const invitationReaders = Array.isArray(inv.edit?.note?.readers)
+        ? inv.edit?.note?.readers
+        : inv.edit?.note?.readers?.param?.const
+      if (!invitationReaders) return true
+
+      return (
+        !excludedInvitations?.includes(inv.id) &&
+        (!invitationReaders.includes('everyone') || note.readers.includes('everyone')) &&
```

Comment:
shouldn't the second condition be `invitationReaders.includes('everyone') && note.readers.includes('everyone')`? if the note contains everyone then the invitation should contain everyone too?

Reply:
that would mean that the only invitations that are shown are the ones that are readable to everyone.

Reply:
OK, I think I fixed the condition now. In order to be shown as a reply invitation it must be:
Not on the exclude list 
AND 
either the note is readable by everyone 
or the invitation has readers that aren't const 
or the invitation readers are a subset of the note readers

Reply:
Looks good to me.